src/main/java/org/jitsi/stats/DataDogStatsClient.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright @ 2018 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.stats;
+
+import com.timgroup.statsd.*;
+import org.jitsi.util.*;
+
+/**
+ * Encapsulates a {@link NonBlockingStatsDClient} to be able to send
+ * statistics to local data-hog server
+ *
+ * @author Nik Vaessen
+ */
+public class DataDogStatsClient
+{
+    /**
+     * The logger of this class
+     */
+    private static final Logger logger
+        = Logger.getLogger(DataDogStatsClient.class);
+
+    /**
+     * Single instance of this {@link DataDogStatsClient}
+     */
+    private static DataDogStatsClient singleton = new DataDogStatsClient();
+
+    /**
+     * The client to the data-hog service
+     */
+    private NonBlockingStatsDClient innerClient;
+
+    /**
+     * Create the single instance of this {@link DataDogStatsClient}
+     * by creating the {@link NonBlockingStatsDClient}
+     */
+    private DataDogStatsClient()
+    {
+        this.innerClient = new NonBlockingStatsDClient(
+            "jicofo",
+            "localhost",
+            8125
```

Comment:
Let's pass all of these as parameters ("localhost" and 8125 could be default values). We didn't do that in jibri only because we were rushing